### PR TITLE
customer schema with Authentcation

### PR DESCRIPTION
**Pull request by hindtak**

### CODE REVIEW

File: server/middlewares/verifyJwtCus.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const verifyCus = (req, res, next) => {
```
Comment (Samireddebbarhi) [inline server/middlewares/verifyJwtCus.js:1]: you should rename a declaration variable of verifyCus to verifyJwtCustomer as an example

File: server/Models/Customer.js [hunk 2]
```diff
@@ -1,2 +1,27 @@
...
+    },
+    email: {
+      type: String,
+      required: true,
+    },
+    password: {
+      type: String,
+      required: true,
+    },
+    permiumstatus: {
```
Comment (Samireddebbarhi) [inline server/Models/Customer.js:20]: To make  a clean Code ,It's better to write as premiumStatus or premium_status

File: server/Models/Customer.js [hunk 3]
```diff
@@ -1,2 +1,27 @@
...
-console.log("helloo ")
\ No newline at end of file
+const mongoose = require("mongoose");
+const SchemaCustomer = new mongoose.Schema({
+    username: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    dateofbirth: {
```
Comment (Samireddebbarhi) [inline server/Models/Customer.js:8]: the same thing

File: server/.env [hunk 4]
```diff
@@ -1,2 +1,4 @@
-URI=mongodb+srv://samired:Administer24@app.0cmtipn.mongodb.net/?retryWrites=true&w=majority&appName=App
-token_Access=f173dde08cc54b448526
\ No newline at end of file
+URI=mongodb+srv://hind_user:admin02@app.0cmtipn.mongodb.net/?retryWrites=true&w=majority&appName=App
+token_Access=f173dde08cc54b448526
+TOKEN_SECRET=0763b32c5f1f167b8858594a1e37e044184b2225ee851c4fd03d0e7e752cd2af
+# eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI2NWU0OWIyMGM3MDUxODgzMmYxYWVhZjQiLCJpYXQiOjE3MDk1NDk4NDEsImV4cCI6MTcwOTU2MDY0MX0.8t1ARs_Ndtw_iizC8pK483Hmz7o6kD3dXp8QXa9DqEU
```
Comment (Samireddebbarhi) [inline server/.env:4]: Delete this comment

**Review by Samireddebbarhi — COMMENTED** [review]
reviewed !!

File: server/middlewares/verifyJwtCus.js (hunk 1 above)
Comment (hindtak) [reply server/middlewares/verifyJwtCus.js:1]: ok , i will fix it

File: server/Models/Customer.js (hunk 2 above)
Comment (hindtak) [reply server/Models/Customer.js:20]: Not Well

File: server/.env (hunk 4 above)
Comment (hindtak) [reply server/.env:4]: Okey

**Pull request closed without merging**